packages/zkdb/src/sdk/impl/collection.ts

```diff
@@ -171,14 +171,22 @@ export class CollectionQueryImpl implements ZKCollection {
       serialize: () => DocumentEncoded;
     },
   >(model: InstanceType<T>, permissions?: Permissions): Promise<MerkleWitness> {
-    const documentPermission =
-      permissions ?? (await this.getOwnership()).permissions;
+    const isPermissionNotPassed =
+      !permissions || Object.keys(permissions).length === 0;
+
+    const documentPermission = isPermissionNotPassed
+      ? await this.getOwnership()
+      : permissions;
 
     const result = await this.apiClient.doc.create({
       databaseName: this.databaseName,
       collectionName: this.collectionName,
       documentRecord: (model as any).serialize(),
-      documentPermission,
+      documentPermission: {
```

Comment:
if documentPermission must contain 3 fields permissionOwner permissionOther permissionGroup had be 
`const isPermissionPassed = permissions && Object.keys(permissions).length === 3;`